### PR TITLE
Updating to 0.0.9

### PR DESCRIPTION
**Pull request by AppleEducate**
- Fixing Android Gradle for Latest Version

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -12,15 +12,11 @@ class MyApp extends StatefulWidget {
...
-
   @override
   initState() {
     super.initState();
-    StripeSource.setPublishableKey("pk_test");
-    StripeSource.addSource().then((String token) {
-      _addSource(token);
-    });
+
+    StripeSource.setPublishableKey("sk_0G1JZIKlGks6b8DhZUf5vfJY7XCMq");
```
Comment (jonasbark) [inline example/lib/main.dart]: please revert this... :)

Comment (AppleEducate) [reply example/lib/main.dart]: oh snap, thanks that definitely wasn't supposed to be there

Comment (jonasbark) [reply example/lib/main.dart]: I'll squash while merging 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [0.0.9] - Updating Dependency
```
Comment (jonasbark) [inline CHANGELOG.md]: mind adding the date instead?

File: example/lib/main.dart (hunk 1 above)
Comment (AppleEducate) [reply example/lib/main.dart]: It was a test key and not a public one FYI, but I still needed to remove it

File: CHANGELOG.md (hunk 2 above)
Comment (AppleEducate) [reply CHANGELOG.md]: Yep!

**Comment by jonasbark** [comment]
Thanks for the contribution!

**Comment by AppleEducate** [comment]
Absolutely!

**Pull request merged**